article.tex

```diff
@@ -75,34 +75,34 @@
 
 
 \section[Introduction]{Introduction}
-Before each statistical analysis, data cleaning is often required to ensure a good data quality.
-Data cleaning is the process of detecting errors and inconsistencies in data sets \citep{Chapman_2005}.
+Before each statistical analysis, data cleaning is often required to ensure good data quality.
+Data cleaning is the process of detecting errors and inconsistencies in datasets \citep{Chapman_2005}.
 In practice, the data cleaning step is often more time consuming than the subsequent statistical analysis, particularly, when the analysis relies on multiple data sources that need to be joined.
 
-When dealing with chemical datasets (e.g. environmental monitoring data, toxicological data), a first step is often to validate the names of chemicals or to link them to unique codes that simplify subsequent querying and appending of compound-related physico-chemical or toxicological information.
+When dealing with chemical datasets (e.g.\ environmental monitoring data, toxicological data), a first step is often to validate the names of chemicals or to link them to unique codes that simplify subsequent querying and appending of compound-related physico-chemical or toxicological information.
 Several web sources provide chemical names or link them to unique codes (see also section \emph{Data sources} below).
 However, manual searching for each compound, often through a graphical web interface, is tedious, error-prone and not reproducible \citep{Peng_2009}.
 
-To simplify, robustify and automate this task, i.e. to search and retrieve chemical information from the web, we created the \pkg{webchem} package for the free and open source \proglang{R} language \citep{r_2015, Wehrens_2011}.
+To simplify, robustify and automate this task, i.e.\ to search and retrieve chemical information from the web, we created the \pkg{webchem} package for the free and open source \proglang{R} language \citep{r_2015, Wehrens_2011}.
```

Comment:
By default, LaTeX adds extra space after each fullstop (intended for sentence periods). However, this is not wanted when using abbreviations. Therefore, you may either escape the subsequent space or toggle off the extra spacing in general using \frenchspacing in the preamble (Source: https://en.wikibooks.org/wiki/LaTeX/Tips_and_Tricks#id_est_and_exempli_gratia_.28i.e._and_e.g..29)
But maybe you can leave this to the JSS typesetters.